share/completions/pacman.fish

```diff
@@ -136,3 +136,4 @@ complete -c $progname -n "$files" -l machinereadable -d 'Show in machine readabl
 # Using "pkg.tar.*" here would change __fish_complete_suffix's descriptions to "unknown"
 complete -c $progname -n "$upgrade" -xa '(__fish_complete_suffix pkg.tar.xz)' -d 'Package file'
 complete -c $progname -n "$upgrade" -xa '(__fish_complete_suffix pkg.tar.gz)' -d 'Package file'
+complete -c $progname -n "$upgrade" -xa '(__fish_complete_suffix pkg.tar)' -d 'Package file'
```

Comment:
Actually, all these can be supplanted with a single call to `__fish_complete_suffix`, reducing IO. `__fish_complete_suffix` accepts expressions, so you can just use `__fish_complete_suffix pkg.tar{,.xz,.gz}` to accomplish the same, only faster.

Reply:
Also, is that supposed to be `.pkg[.....]` with a leading `.`?

Reply:
Yes, it probably should be `.pkg.whatever`. I'll change it to the single expression then.

Reply:
And I made them into a single call. For some reason the curly braces need to be escaped to work though.